main.cpp

```diff
@@ -63,9 +55,58 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
+     //int, double, char, bool, float, void, unsigned int
+
+    int myFirstInt = -1;
+    int mySecondInt = 22000;
+    int myThirdInt = -30526;
+
+    double myFirstDouble = 1.14;
+    double mySecondDouble = 155.35;
+    double myThirdDouble = 33.58;
+
+    float myFirstFloat = 8;
+    float mySecondFloat = 17;
+    float myThirdFloat = 160;
```

Comment:
While it is perfectly safe to initialize `float`'s with `int`'s (that's what you're doing here), I would like you to be explicit and initialize your `float`'s with floats on the right side of the `=`.
- [ ] FIXME replace `int`s with `float`s on right side of `=`